lib/rule_blocklist.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const a = "a"
+  , Config = require("eslint/lib/config")
+  , merge = require("eslint/lib/config/config-ops").merge
+  ;
+
+const blocklistedRules = {
+  "no-unresolved": "Code Climate doesn't fetch your project's dependencies so this rule produces high amount of false positives."
+};
```

Comment:
We should definitely audit all rules currently available via our vendored plugins and add any that rely on module resolution to this blocklist either in this PR or the next one.

Here's one example of another rule that should be in the blocklist:
`import/extensions`
https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/extensions.js

> Here are a few examples of other reasons we may want to disable a rule I can think of

Until we actually want to ignore rules for another reason, I'd stick with a simpler implementation that doesn't map particular rules to warning messages. I don't think it's worth adding now. If we need it later, we can add it then.